packages/ui/src/common/hooks/useSignAndSendTransaction.ts

```diff
@@ -37,31 +40,32 @@ const observeTransaction = (
     }
 
     if (status.isInBlock) {
-      info('Included at block hash', JSON.stringify(status.asInBlock))
+      const hash = status.asInBlock
+      info('Included at block hash', JSON.stringify(hash))
       info('Events:')
       info(JSON.stringify(events))
+      setBlockHash(hash.toString())
 
-      send({
-        type: hasErrorEvent(events) ? 'ERROR' : 'FINALIZING',
-        events,
-        fee,
-      })
+      send(hasErrorEvent(events) ? 'ERROR' : 'FINALIZING')
 
       setPending(false)
     }
 
     if (status.isFinalized) {
       send({
-        type: hasErrorEvent(events) ? 'ERROR' : 'SUCCESS',
+        type: hasErrorEvent(events) ? 'ERROR' : 'PROCESSING',
         events,
         fee,
       })
     }
   }
 
-  const errorHandler = () => send({ type: 'ERROR', events: [] })
+  const errorHandler = () => send({ type: 'CANCELED', events: [] })
 
-  transaction.subscribe(statusCallback, errorHandler)
+  transaction.subscribe({
+    next: statusCallback,
+    error: errorHandler,
+  })
```

Comment:
Update deprecated signature of the subscribe method:
https://rxjs.dev/deprecations/subscribe-arguments